base/strings/util.jl

```diff
@@ -482,6 +482,25 @@ julia> replace("The quick foxes run quickly.", r"fox(es)?" => s"bus\\1")
 replace(s::AbstractString, pat_f::Pair; count=typemax(Int)) =
     replace(String(s), pat_f, count=count)
 
+
+"""
+    replace(s::AbstractString, reps::Pair...; count::Integer=typemax(Int))
+
+perform a successive series of `replace` operations on `s`.
```

Comment:
It would be good to be more explicit about the fact that the replacements are processed in the order specified. Also this should be capitalized as a sentence.